db/prisma/migrations/20230721094123_remove_pano_post_slug/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
since we are using planet scale is this needed anymore ? i thought they are just using push on all changes so migrations are not needed. I might be wrong, don't know planet scale that much

Reply:
yeah this is mainly for local dev